app/models/question.rb

```diff
@@ -22,4 +22,8 @@ class Question < ActiveRecord::Base
   validates :number, presence: true, numericality: { only_integer: true }
 
   validates :question_type, presence: true, inclusion: {in: %w[short_answer multiple_choice true_false]}
+
+  def format_as_sms
+    return self.text + "\n" + self.response_choices.map{ |r| r.key + ") " + r.text}.join("\n")
```

Comment:
Would be a little faster if you use string interpolation instead of concatenation `response_choices.map{ |r| "#{r.key}) #{r.text}" }`. You might even be able to do this on the whole line, I'm not sure about the syntax for that though.